art-cluster/cloudflare/src/index.ts

```
@@ -26,10 +26,35 @@ async function secureProxyToValidatedEndpoint(request: Request, targetBaseUrl: s
-    // Create final URL by appending path to secure endpoint
-    const finalUrl = secureEndpoint + remainingPath;
 
+    // Sanitize remainingPath to prevent SSRF via path traversal or authority injection
+    let decodedPath: string;
+    try {
+        decodedPath = decodeURIComponent(remainingPath);
+    } catch {
+        return new Response('Invalid path', { status: 400 });
+    }
+    const segments = decodedPath.split('/');
+    if (segments.some((s) => s === '.' || s === '..') || /[@\\\u0000-\u001F\u007F]/.test(decodedPath)) {
```

Biome (2.3.14)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)